lib/DDG/Spice/Currency.pm

```diff
@@ -33,7 +33,7 @@ my $number_re = number_style_regex();
 my $cardinal_re = join(' |', qw(hundred thousand k million m billion b trillion)).' ';
 my $from_qr = qr/(?<fromSymbol>\p{Currency_Symbol})|(?:(?<from>$currency_qr)s?)/;
 my $amount_qr = qr/(?<amount>$number_re*)\s?(?<cardinal>$cardinal_re)?/;
-my $keyword_qr = qr/(?:\s?(?<currencyKeyword>(?:currency|value))\s?)/i;
+my $keyword_qr = qr/(?:\s?(?<currencyKeyword>(?:currency|value|price|))\s?)/i;
```

Comment:
@adityatandon007 there shouldn't be a trailing `|` after `|price`. This dramatically changes what the regex matches on, so please remove that 👍